opm/parser/eclipse/Parser/tests/ParserTests.cpp

```diff
@@ -298,6 +299,7 @@ BOOST_AUTO_TEST_CASE( PATHS_with_backslashes ) {
 
     parseContext.update( ParseContext::PARSE_MISSING_INCLUDE , Opm::InputError::THROW_EXCEPTION);
     parser.parseFile( "testdata/parser/PATHSWithBackslashes.data", parseContext );
+    BOOST_CHECK(parser.hasKeyword("OIL"));
```

Comment:
Ehhh - this just tests that the parser instance recognizes the `OIL` keyword, not that it has actually loaded that keyword into a `Deck` object. The test you need is:

```C++
const auto deck = parser.parseFile( ... );
BOOST_CHECK( deck.hasKeyword("OIL"));
```